pkg/ingester/stream.go

```diff
@@ -8,7 +8,10 @@ import (
 	"sync"
 	"time"
 
-	"github.com/cortexproject/cortex/pkg/ingester/client"
+	"github.com/cortexproject/cortex/pkg/util"
+
```

Comment:
can you pack everything please ?

Reply:
I leave imports formatting to Goland and goimports, so I'll try, but I won't fight them if they decide otherwise.